es-bs-base/scss/_variables.scss

```diff
@@ -34,6 +34,8 @@ $grays: map-merge(
 );
 
 $blue:    #275ba9 !default;
+$lightblue: #8cd1ff;
+$grayblue:  #c9e9f6;
```

Comment:
Think it might be best to always include `!default` on variables in this file.

Reply:
So that verticals can override? I guess that makes sense.